Added language tags

MeuhV1 opened the pull request.

Commit:
Added language tags

MeuhV1 commented:
I replaced all "No Translation" texts with the English text. Only for non-pedia text entries.

I used a script to auto-replace everything. It should have replaced only the new tags I created in the first commit.

MeuhV1 commented:
Added translation for vanilla techs and quotes

dguenms commented on Assets/XML/Text/DynamicNames/DynamicNames_Arabia.xml:
```diff
@@ -304,14 +340,26 @@
 	<TEXT>
 		<Tag>TXT_KEY_CIV_ARABIAN_NAME_SPAIN</Tag>
 		<English>Esbanya</English>
+		<French>banya</French>
```
Looks like something went wrong here.

dguenms commented on Assets/XML/Text/DynamicNames/DynamicNames_Arabia.xml:
```diff
@@ -340,6 +388,10 @@
 	<TEXT>
 		<Tag>TXT_KEY_CIV_ARABIAN_MUGHALS</Tag>
 		<English>Emirate of Delhi</English>
+		<French>mirate of Delhi</French>
```
Here too.

dguenms commented on Assets/XML/Text/DynamicNames/DynamicNames_Byzantium.xml:
```diff
@@ -36,70 +44,138 @@
 	<TEXT>
 		<Tag>TXT_KEY_CIV_BYZANTIUM_EPIRUS</Tag>
 		<English>Epirus</English>
+		<French>pirus</French>
```
.

dguenms commented on Assets/XML/Text/DynamicNames/DynamicNames_Byzantium.xml:
```diff
@@ -36,70 +44,138 @@
 	<TEXT>
 		<Tag>TXT_KEY_CIV_BYZANTINE_EGYPT</Tag>
 		<English>Exarchate of Alexandria</English>
+		<French>xarchate of Alexandria</French>
```
.

dguenms commented on Assets/XML/Text/DynamicNames/DynamicNames_Byzantium.xml:
```diff
@@ -36,70 +44,138 @@
 	<TEXT>
 		<Tag>TXT_KEY_CIV_BYZANTINE_BABYLONIA</Tag>
 		<English>Exarchate of Ctesiphon</English>
+		<French>xarchate of Ctesiphon</French>
```
.

dguenms commented on Assets/XML/Text/DynamicNames/DynamicNames_Byzantium.xml:
```diff
@@ -36,70 +44,138 @@
 	<TEXT>
 		<Tag>TXT_KEY_CIV_BYZANTINE_GREECE</Tag>
 		<English>Exarchate of Achaia</English>
+		<French>xarchate of Achaia</French>
```
.

dguenms commented on Assets/XML/Text/DynamicNames/DynamicNames_Byzantium.xml:
```diff
@@ -36,70 +44,138 @@
 	<TEXT>
 		<Tag>TXT_KEY_CIV_BYZANTINE_CARTHAGE</Tag>
 		<English>Exarchate of Africa</English>
+		<French>xarchate of Africa</French>
```
.

dguenms commented on Assets/XML/Text/DynamicNames/DynamicNames_Byzantium.xml:
```diff
@@ -36,70 +44,138 @@
 	<TEXT>
 		<Tag>TXT_KEY_CIV_BYZANTINE_PERSIA</Tag>
 		<English>Exarchate of Parthia</English>
+		<French>xarchate of Parthia</French>
```
.

dguenms commented on Assets/XML/Text/DynamicNames/DynamicNames_Byzantium.xml:
```diff
@@ -36,70 +44,138 @@
 	<TEXT>
 		<Tag>TXT_KEY_CIV_BYZANTINE_ROME</Tag>
 		<English>Exarchate of Ravenna</English>
+		<French>xarchate of Ravenna</French>
```
.

dguenms commented on Assets/XML/Text/DynamicNames/DynamicNames_Byzantium.xml:
```diff
@@ -36,70 +44,138 @@
 	<TEXT>
 		<Tag>TXT_KEY_CIV_BYZANTINE_SPAIN</Tag>
 		<English>Exarchate of Spania</English>
+		<French>xarchate of Spania</French>
```
.

dguenms commented on Assets/XML/Text/DynamicNames/DynamicNames_Carthage.xml:
```diff
@@ -4,23 +4,38 @@
 			<Gender>Neuter:Neuter:Neuter</Gender>
 			<Plural>0:0:0</Plural>
 		</German>
+		<Italian>No Translation</Italian>
```
Should use English text.

dguenms commented on Assets/XML/Text/DynamicNames/DynamicNames_China.xml:
```diff
@@ -69,6 +81,7 @@
 			<Gender>Female</Gender>
 			<Plural>1</Plural>
 		</French>
+		<German>No Translation</German>
```
Should use English text.

dguenms commented on Assets/XML/Text/DynamicNames/DynamicNames_Colombia.xml:
```diff
@@ -40,21 +52,41 @@
 	<TEXT>
 		<Tag>TXT_KEY_CIV_COLOMBIA_EMPIRE</Tag>
 		<English>Empire of Gran Colombia</English>
+		<French>mpire of Gran Colombia</French>
```
.

dguenms commented on Assets/XML/Text/DynamicNames/DynamicNames_Japan.xml:
```diff
@@ -81,6 +81,7 @@
 			<Gender>Male</Gender>
 			<Plural>0</Plural>
 		</French>
+		<German>No Translation</German>
```
Should use English text.

dguenms commented on Assets/XML/Text/DynamicNames/DynamicNames_Mexico.xml:
```diff
@@ -33,6 +37,7 @@
 			<Gender>Male</Gender>
 			<Plural>1</Plural>
 		</French>
+		<German>No Translation</German>
```
Should use English text.

dguenms commented on Assets/XML/Text/DynamicNames/DynamicNames_Moors.xml:
```diff
@@ -100,26 +124,50 @@
 	<TEXT>
 		<Tag>TXT_KEY_CIV_MOORISH_ARABIA</Tag>
 		<English>East Umayyad Caliphate</English>
+		<French>ast Umayyad Caliphate</French>
```
.

dguenms commented on Assets/XML/Text/DynamicNames/DynamicNames_Ottomans.xml:
```diff
@@ -116,10 +132,18 @@
 	<TEXT>
 		<Tag>TXT_KEY_CIV_OTTOMAN_BABYLONIA</Tag>
 		<English>Eyalet of Mesopotamia</English>
+		<French>yalet of Mesopotamia</French>
```
.

dguenms commented on Assets/XML/Text/DynamicNames/DynamicNames_Ottomans.xml:
```diff
@@ -172,6 +196,10 @@
 	<TEXT>
 		<Tag>TXT_KEY_CIV_OTTOMAN_PHOENICIA</Tag>
 		<English>Eyalet of Syria</English>
+		<French>yalet of Syria</French>
```
.

dguenms commented on Assets/XML/Text/DynamicNames/DynamicNames_Ottomans.xml:
```diff
@@ -200,6 +228,10 @@
 	<TEXT>
 		<Tag>TXT_KEY_CIV_OTTOMAN_BYZANTIUM</Tag>
 		<English>Eyalet of R&#251;m</English>
+		<French>yalet of R&#251;m</French>
```
.

dguenms commented on Assets/XML/Text/DynamicNames/DynamicNames_Ottomans.xml:
```diff
@@ -228,10 +260,18 @@
 	<TEXT>
 		<Tag>TXT_KEY_CIV_OTTOMAN_RUSSIA</Tag>
 		<English>Eyalet of Yedisan</English>
+		<French>yalet of Yedisan</French>
```
.

dguenms commented on Assets/XML/Text/DynamicNames/DynamicNames_Ottomans.xml:
```diff
@@ -228,10 +260,18 @@
 	<TEXT>
 		<Tag>TXT_KEY_CIV_OTTOMAN_INDONESIA</Tag>
 		<English>Eyalet of Aceh</English>
+		<French>yalet of Aceh</French>
```
.

dguenms commented on Assets/XML/Text/External/AutoSave.xml:
```diff
@@ -5,49 +5,65 @@
 	<TEXT>
 		<Tag>TXT_KEY_BUG_OPT_AUTOSAVE__CREATEENDSAVE_TEXT</Tag>
 		<English>End Save</English>
+		<French>d Save</French>
```
.

dguenms commented on Assets/XML/Text/External/AutoSave.xml:
```diff
@@ -5,49 +5,65 @@
 	<TEXT>
 		<Tag>TXT_KEY_BUG_OPT_AUTOSAVE__CREATEEXITSAVE_TEXT</Tag>
 		<English>Exit Save</English>
+		<French>xit Save</French>
```
.

dguenms commented on Assets/XML/Text/External/BUFFY.xml:
```diff
@@ -11,155 +11,209 @@
 		<French>Activ&#233;</French>
 		<German>Aktivieren</German>
 		<Italian>Attivato</Italian>
+		<Spanish>abled</Spanish>
```
.

dguenms commented on Assets/XML/Text/External/Event Signs.xml:
```diff
@@ -13,14 +13,18 @@
 	<TEXT>
 		<Tag>TXT_KEY_BUG_OPT_EVENTSIGNS__ENABLED_TEXT</Tag>
 		<English>Event Signs</English>
+		<French>vent Signs</French>
```
.

dguenms commented on Assets/XML/Text/External/Reminder.xml:
```diff
@@ -8,47 +8,54 @@
 		<French>Activer les Rappels [ALT + M]</French>
 		<German>Erinnerungen aktivieren</German>
 		<Italian>Abilita i Promemoria [ALT + M]</Italian>
+		<Spanish>able Reminders [ALT + M]</Spanish>
```
.

dguenms commented on Assets/XML/Text/External/Strategy Overlay.xml:
```diff
@@ -10,140 +10,185 @@
 		<French>Activ&#233;</French>
 		<German>Aktivieren</German>
 		<Italian>Attivata</Italian>
+		<Spanish>abled</Spanish>
```
.

dguenms reviewed (changes requested):
Some English texts should have been used.

There is also an issue with texts starting with a capital E, seems like the regex did not capture those correctly. I only reviewed diffs small enough to show up, so you should probably do a search for "<English>E" and verify for the entire diff.

MeuhV1 commented:
It seems my script removed the first letters of all entries starting with an E.

Fixed it.

dguenms reviewed (commented):
There are still some comments open where the English text should be used.

Commit:
Translations for vanilla techs
Translations for various things (copied from RFCE)
English text for all tags that don't have a translation (except pedia texts)

MeuhV1 commented:
Replaced all "No translations" text with the English texts. (Except pedia texts, they still have the "No translation" text)

Merged all "translation" commits into 1 commit.

dguenms reviewed (approved):
If you remove the recent conflicts I can merge this.

MeuhV1 commented:
Done.
